vidur/config/node_sku_config.py

```diff
@@ -0,0 +1,94 @@
+from dataclasses import dataclass, field
+
+from vidur.config.base_fixed_config import BaseFixedConfig
+from vidur.logger import init_logger
+from vidur.types import NodeSKUType, DeviceSKUType
+
+logger = init_logger(__name__)
+
+
+@dataclass
+class BaseNodeSKUConfig(BaseFixedConfig):
+    num_devices_per_node: int = field(
```

Comment:
nit: we don't need to define help strings for node and model config anymore?

Reply:
Yeah right, we don't need them anymore. Removing helper strings.